cli/flox/src/utils/init/sentry.rs

```diff
@@ -47,7 +47,7 @@ pub fn init_sentry() -> Option<ClientInitGuard> {
         send_default_pii: false,
 
         // Enable debug mode when needed
-        debug: true,
+        debug: false,
```

Comment:
Suggest we tie this to command line verbosity... we may WANT to see this sometimes.  We could tie it to `-vv` to limit it's output to trace level.